lib/davinci_pdex_test_kit/pdex_payer_server_suite.rb

```diff
@@ -159,20 +159,20 @@ class PDexPayerServerSuite < Inferno::TestSuite
 
           input_order :url, :credentials, :no_member_match_request, :multiple_member_match_request
 
-          group from: :pdex_no_member_matches_group
-          group from: :pdex_multiple_member_matches_group
+          group from: :pdex_no_member_matches
+          group from: :pdex_multiple_member_matches
         end
 
         group do
           title 'PDEX Search and Read API (US Core plus additional PDex resource types)' 
-          id :pdex_search_and_read_api_coverage
+          id :pdex_fhir_api_coverage
           
-          group from: :pdex_explanation_of_benefit_group
+          group from: :pdex_eob
 
           # Import all US Core v3.1.1 groups without the Suite
           USCoreTestKit::USCoreV311::USCoreTestSuite.groups[1].groups.each do |group|
             # This prevents a second OAuth credentials box from appearing in UI
-            group(from: group.ancestors[1].id)
+            group(from: group.ancestors[1].id, id: group.ancestors[1].id.delete_prefix('us_core_v311_'))
```

Comment:
Deleting this prefix may cause a conflict if we ever decide to implement pdex 2.1 which uses us core 6.1.0. However AFAIK we have no plans to do this, so might as well delete the prefix for now.

Reply:
Also hopefully this doesn't cause any id conflict with any other test kit that gets added to inferno platform. Maybe add `pdex_` prefix to the all the ids?

Reply:
Deleting the prefix is safe, because the id is still going to be appended to the suite id.